docs/refs/api/catalogd-webserver.md

```diff
@@ -0,0 +1,88 @@
+# catalogd webserver
+
+[catalogd](https://github.com/operator-framework/catalogd), the OLM v1 component for making catalog contents available on cluster, includes
```

Comment:
```suggestion
[Catalogd](https://github.com/operator-framework/catalogd), the OLM v1 component for making catalog contents available on cluster, includes
```

Reply:
Addressed in https://github.com/operator-framework/operator-controller/pull/1331/commits/de1767ea0edfa05872092fc0aee3dda0616bd305